MediaGalleryUi/Test/Mftf/ActionGroup/AssertEnhancedMediaGalleryImageDeletedActionGroup.xml

```diff
@@ -8,14 +8,14 @@
 
 <actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
               xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
-    <actionGroup name="AssertEnhancedMediaGalleryImageDeletedActionGroup">
+    <actionGroup name="AssertAdminEnhancedMediaGalleryImageDeletedActionGroup">
```

Comment:
@konarshankar07 thanks for the updates! Please rename the file according to the action group name change